src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -499,7 +475,25 @@ public static String convertToStandardDuration(Long timeInMilliseconds) {
     }
 
     /**
-     * Parse a `LocalDateTime` object from separated date, hour and minute strings.
+     * Parses a {@code LocalDateTime} object from a date time string according to a pattern.
+     * Returns {@code null} on error.
+     */
+    public static LocalDateTime parseLocalDateTime(String dateTimeString, String pattern) {
```

Comment:
At first I thought I would need this function in AdminActivityLogPageData so I refactored it out here. In the end I didn't need it, but I think  it's still good to prevent repetition of code.